src/components/Footer.tsx

```diff
@@ -1,11 +1,10 @@
-import "./Footer.styled";
 import { StyledFooter, StyledText } from "./Footer.styled";
 
 interface FooterProps {
 	numUsers: number;
 }
 
-const Footer: React.FC<FooterProps> = ({ numUsers }) => {
+const Footer: React.FC<FooterProps> = ({ numUsers }: { numUsers: number }) => {
```

Comment:
```suggestion
const Footer = ({ numUsers }: FooterProps ) : JSX.Element => {
```
While you are updating this line, would you mind changing the structure of this function declaration to be consistent with all the other files?